Cargo.toml

```
@@ -15,5 +15,7 @@ include = [
     "Cargo.toml",
 ]
 
-[dependencies]
-nom = "5"
+[dependencies.nom]
+version = "5.1.1"
```

I'd recommend to use more relax version format, like `5.1` or just `5`, in case it was a feature of 5.0.x.

---

I don't understand what you mean. `std`, `alloc` and `lexical` are all definitely cargo features of nom 5.1.1 (see [Cargo.toml](https://github.com/Geal/nom/blob/5.1.1/Cargo.toml)). On the other hand, making sure this actually works against an older 5.x nom is harder (in CI and such) because cargo will default to using the latest compatible version that's available.

---

I mean that cargo will use latest version of the crate anyway, if you set just `5`(it has all the features). However if some other dependency of downstream crate strictly depends on 5.0 you won't have duplicate dependencies (5.1.1 and 5.0, like it'll be now)

---

Not sure why anybody would do that, but sure, I'll change it back.

```suggestion
version = "5"
```